### PR TITLE
Fix FeatSelWrapper to work with bits (#2468)

### PR DESCRIPTION
**Pull request by jakob-r**
Also cleant up the code a bit
Changes: See NEWS file

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -16,13 +16,18 @@
 * praznik.CMIM
 
 ## filter - general
-* Replaced filters `information.gain`, `gainratio` and `symmetrical.uncertainty`depending on package `FSelector` by package `FSelectorRcpp`. 
+* Replaced filters `information.gain`, `gainratio` and `symmetrical.uncertainty`depending on package `FSelector` by package `FSelectorRcpp`.
```
Comment (larskotthoff) [inline NEWS.md]: This shouldn't be in this PR.

File: R/FeatSelResult.R [hunk 2]
```diff
@@ -20,37 +20,33 @@
...
-    x = c(head(x, printed.features), "...")
-  else
-    x = head(x, printed.features)
+  shortenX = function(x) {
+    clipString(collapse(x, ", "), 50L)
+  }
 
-  catf("Features (%i): %s", n.feats, collapse(x$x, ", "))
+  if (!all.equal(x$x.bit.names, x$x)) {
+    catf("Bits (%i): %s", length(x$x.bit.names), shortenX(x$x.bit.names))
```
Comment (larskotthoff) [inline R/FeatSelResult.R]: Shouldn't that be `messagef`?

Comment (berndbischl) [reply R/FeatSelResult.R]: no thats a printer?

Comment (jakob-r) [reply R/FeatSelResult.R]: catf because a massage is not the same as cat.

File: NEWS.md (hunk 1 above)
Comment (jakob-r) [reply NEWS.md]: I changed it back.

**Comment by larskotthoff** [comment]
Looks good, but I'll wait until Travis is happy again before merging.

**Pull request merged**